README.md

```diff
@@ -219,6 +219,29 @@ To configure chamber to use the S3 backend, use `chamber -b s3 --backend-s3-buck
 
 This feature is experimental, and not currently meant for production work.
 
+### S3 Backend using KMS Key Encryption (Experimental)
+
+This backend is similar to the S3 Backend but uses KMS Key Encryption to encrypt your documents at rest, similar to the SSM Backend which encrypts your secrets at rest. You can read how S3 Encrypts documents with KMS [here](https://docs.aws.amazon.com/kms/latest/developerguide/services-s3.html).
+
+The highlights of SSE-KMS are:
+- You can choose to create and manage encryption keys yourself, or you can choose to use your default service key uniquely generated on a customer by service by region level.
+- The ETag in the response is not the MD5 of the object data.
+- The data keys used to encrypt your data are also encrypted and stored alongside the data they protect.
+- Auditable master keys can be created, rotated, and disabled from the AWS KMS console.
+- The security controls in AWS KMS can help you meet encryption-related compliance requirements.
+Source https://docs.aws.amazon.com/AmazonS3/latest/dev/UsingKMSEncryption.html
+
+To configure chamber to use the S3 backend, use `chamber -b s3-kms --backend-s3-bucket=mybucket`.  You must also supply an environment variable of the KMS Key Alias to use CHAMBER_KMS_KEY_ALIAS, by default "alias/parameter_store_key" will be used.
```

Comment:
How would you feel about making this a flag instead? I just think being explicit with these security details is better. I realize it doesn't mesh with the existing usage of `CHAMBER_KMS_KEY_ALIAS`, but I think as our scope grows, the "simplicity" of env vars over flags diminishes.

Reply:
I really like the Environment Variable interface, we run a series of different Docker containers of different security domains that we pass in CHAMBER_KMS_KEY_ALIAS into to dynamically write secrets with different keys. In practice, we could pass in the key alias and then invoke the command with the flag set to the environment variable.

Would you consider both interfaces? It can explode if you provide both the flag and the environment variable?

Reply:
Yep, happy to have both, similar to how `--backend/CHAMBER_BACKEND` works.